### PR TITLE
Fix missing closing tag.

### PR DESCRIPTION
**Pull request by aadcg**

### CODE REVIEW

**Comment by aadcg** [comment]
I've now realized that non closed tags generate valid HTML.

Therefore I believe I must close this PR.

Unrelated: does spinneret provide a way to force closing tags?  Thanks.

**Comment by ruricolist** [comment]
That can be done with *html-style*.

**Comment by aadcg** [comment]
Thanks @ruricolist!

See the changes that I propose, in light of the insights.

File: README.md [hunk 1]
```diff
@@ -132,6 +132,9 @@ nil, output is verbose but predictable:
           (:p "Text " (:a "link text") " more text"))))
     => "<div><p>Text <a>link text</a>  more text</p></div>"
 
+Notice that bounding `*html-style*` to `:tree` ensures that all tags are
```
Comment (ruricolist) [inline README.md:135]: "bounding" should be "binding"

**Review by ruricolist — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (aadcg) [reply README.md]: I've fixed it, thanks.

**Review by ruricolist — APPROVED** [review]

**Pull request merged**